desc/vmec.py

```diff
@@ -545,7 +557,9 @@ def save(cls, eq, path, surfs=128, verbose=1):  # noqa: C901 - FIXME - simplify
             rmns = file.createVariable("rmns", np.float64, ("radius", "mn_mode"))
             rmns.long_name = "sin(m*t-n*p) component of cylindrical R, on full mesh"
             rmns.units = "m"
-        m, n, x_mn = zernike_to_fourier(eq.R_lmn, basis=eq.R_basis, rho=r_full)
+        r1 = np.ones_like(eq.R_lmn)
+        r1[eq.R_basis.modes[:, 1] < 0] *= -1
+        m, n, x_mn = zernike_to_fourier(r1 * eq.R_lmn, basis=eq.R_basis, rho=r_full)
```

Comment:
So the DESC solution always has a positive Jacobian, and the VMEC solution is always supposed to have a negative Jacobian? Am I understanding that correctly?

Reply:
yes. We will always have a positive jacobian, and vmec (as far as I can tell) always has a negative one.